client/blocks/color-scheme-picker/constants.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+export const ColorSchemes = [
+	{
+		label: translate( 'Default' ),
```

Comment:
Absolutely! The way I've done this in the past is wrapping translate calls in a function that takes a `translate`, and call that during the render phase, where we have `this.props.translate`. I found the same approach the other day in AppBanner:

https://github.com/Automattic/wp-calypso/blob/4c598972f35682c959f4c382e9b16c209f3e9ed9/client/blocks/app-banner/utils.js#L19-L47